ci/images/debian10/Dockerfile

```diff
@@ -24,25 +24,17 @@ RUN apt-get install -y gettext-base
 RUN apt-get install -y git rsync unzip jq
 
 # Installs shellcheck.
-RUN curl -fsSL https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.$(uname -m).tar.xz | \
-    tar -xJ && \
-    mv shellcheck*/shellcheck /usr/local/bin && \
-    rm -R shellcheck*
-
-# Install Go.
-RUN ARCH="$(uname -m | sed 's/x86_64/amd64/; s/aarch64/arm64/')" && \
-    curl -fsSL "https://dl.google.com/go/go1.14.3.linux-$ARCH.tar.gz" | tar -C /usr/local -xz
-ENV GOPATH=/gopath
-# Ensures running this image as another user works.
-RUN mkdir -p $GOPATH && chmod -R 777 $GOPATH
-ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
-
-# More stable than go get
-RUN curl -sfL https://install.goreleaser.com/github.com/goreleaser/nfpm.sh | sh
-
-RUN VERSION="$(curl -fsSL https://storage.googleapis.com/kubernetes-release/release/stable.txt)" && \
-    curl -fsSL "https://storage.googleapis.com/kubernetes-release/release/$VERSION/bin/linux/amd64/kubectl" > /usr/local/bin/kubectl \
-    && chmod +x /usr/local/bin/kubectl
+RUN curl -fsSL https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.$(uname -m).tar.xz \
+  | tar -xJ \
+  && mv shellcheck*/shellcheck /usr/local/bin \
+  && rm -R shellcheck*
+
+# Using GitHub release URL is better because now only rely on one service instead of GitHub and install.goreleaser.com
+RUN curl -sSL https://github.com/goreleaser/nfpm/releases/download/v2.3.1/nfpm_2.3.1_Linux_x86_64.tar.gz | tar -C /tmp -zxv nfpm
+
+RUN VERSION="$(curl -fsSL https://storage.googleapis.com/kubernetes-release/release/stable.txt)" \
+  && curl -fsSL "https://storage.googleapis.com/kubernetes-release/release/$VERSION/bin/linux/amd64/kubectl" >/usr/local/bin/kubectl \
```

Comment:
This wasn't introduced by your PR but we should use the detected architecture here as well.